lib/nerves_network/eapol_manager.ex

```diff
@@ -223,7 +224,8 @@ defmodule Nerves.Network.EAPoLManager do
       pid = start_wpa_supervisor(state)
 
       :ok = registry_register(Nerves.WpaSupplicant, state.ifname)
-      %{%{state | supplicant_port: port} | wpa_pid: pid}
+
+       %{%{state | supplicant_port: port} | wpa_pid: pid}
```

Comment:
This should probably be %{state | supplicant_port: port, wpa_pid: pid}

Reply:
both work, I think, but David's way does look cleaner.

Reply:
can one update two items at once?

Reply:
As long as the keys existed beforehand, then Yes. If not, you could do this 
```
state
|> Map.put(:supplicant_port, port)
|> Map.put(:wpa_pid, pid)
```

Reply:
re-wroked.